translate/storage/flatxml.py

```diff
@@ -0,0 +1,201 @@
+# -*- coding: utf-8 -*-
+
+"""Module for handling flat XML files."""
+
+from lxml import etree
+
+from translate.lang import data
+from translate.misc.xml_helpers import getText, namespaced, reindent
+from translate.storage import base
+from translate.storage.placeables import general
+
+
+class FlatXMLUnit(base.TranslationUnit):
+    """A single term in the XML file."""
+
+    element_name = "str"
+    attribute_name = "key"
+    namespace = None
+    rich_parsers = general.parsers
```

Comment:
Is this unused?

Reply:
As with the other one, I guess this is copy&paste from one of the other formats. I'm not sure if it is used indirectly over the base class, but I'll take a look later.

Reply:
It seems to be used here: https://github.com/translate/translate/blob/4718fabd85a42401f7095569613fa6db9df184e6/translate/storage/base.py#L176-L178
And since ResX (where I got this from) overrides it, I thought this was required. But even with it removed all tests pass just fine.

...which is somewhat unsurprising, not being static there. I'll just remove it.